frontend/feat/feature-flags.json

```diff
@@ -12,10 +12,10 @@
     "additional_search_views": {
       "status": {
         "staging": "switchable",
-        "production": "disabled"
+        "production": "switchable"
       },
       "description": "Toggle additional search views for tag/creator/source.",
-      "defaultState": "off",
```

Comment:
I'm a little confused about what the "right" way to toggle the flag on in production is.

We set it to switchable, but default it to on. That means it's basically on in all environments, but an individual could switch it off by visiting `/preferences`, right? In practice that's irrelevant to everyone except devs because `/preferences` isn't linked anywhere.

So this feature flag setting could be simpified by: replacing the staging entry with a single production entry, set to `enabled`, and removing `defaultState` altogether. Thinking of this in stages, should we do this in two steps, first set production to switchable, default state off, and staging to enabled? And then confirm it in staging as a globally enabled preference (rather than a toggled one), and then update `production` to enabled? It would be nice if we could control these using environment variables so that it doesn't take an entire new release for each stage to take effect.

Reply:
In production it seems that "switchable" is essentially "enabled" but with a convenience method for devs to see the disabled state. 

Abstractly, I think the flow for launching frontend features should be:

- Enable the feature by default in staging, switchable but off by default in prod
- Test it in both of those contexts
- Enable it in prod

I think we could use a doc outlining how to prep and ship user-facing frontend (or API, for that matter) features. I'll create issues this week.